quinn/src/platform/unix.rs

```diff
@@ -224,23 +234,38 @@ fn send(io: &mio::net::UdpSocket, transmits: &[Transmit]) -> io::Result<usize> {
                     // Retry the transmission
                     continue;
                 }
-                io::ErrorKind::PermissionDenied => {
-                    // Transmissions can fail with permission errors for example
-                    // due to iptable rules. Those are not fatal errors, since the
-                    // configuration can be dynamically changed.
-                    // In this case we drop the outgoing packets and let higher
-                    // layers retransmit if required.
-                    return Ok(num_transmits);
+                io::ErrorKind::WouldBlock => return Err(e),
+                _ => {
+                    // Other errors are ignored, since they will ususally be handled
+                    // by higher level retransmits and timeouts.
+                    // - PermissionDenied errors have been observed due to iptable rules.
+                    //   Those are not fatal errors, since the
+                    //   configuration can be dynamically changed.
+                    // - Destination unreachable errors have been observed for other
+                    let now = Instant::now();
+                    if now.saturating_duration_since(*last_send_error) > IO_ERROR_LOG_INTERVAL {
+                        *last_send_error = now;
+                        warn!("sendmmsg error: {:?}, transmits: {:?}", e, transmits);
```

Comment:
Do we really want to log the *contents* of the packets being sent? That seems very verbose and uninteresting.

Reply:
yes, the contents are too verbose. However I thought it was interesting to see the src and destination address as well as if e.g. GSO is used. Could either manually log those fields, or add a `Debug` impl for `Transmit` which does not log the contents.

Reply:
My knee-jerk thought is to favor manually logging, probably pulled out into a helper function.

Reply:
It would need at least 2 helper functions to accommodate for `Transmit` vs `[Transmit]` (mmsg case). Not sure whether that's favorable.

Reply:
Added the helper function, after I recognized we only need to log the first `Transmit`s content (which will always be the "offending" one). Also moved the debounce check there, which makes things a bit cleaner